### PR TITLE
update to MariaDB-11.0.2

### PR DESCRIPTION
**Pull request by DimitrisStaratzis**
[sc-33803]

### CODE REVIEW

**Comment by DimitrisStaratzis** [comment]
I expect the macOS CI to fail because of #269

File: mytile/mytile-metadata.cc [hunk 1]
```diff
@@ -330,7 +330,7 @@ std::string tile::build_datetime_string(THD *thd, uint64_t seconds,
   MYSQL_TIME to;
   if (type != MYSQL_TIMESTAMP_DATE) {
     thd->variables.time_zone->gmt_sec_to_TIME(&to, seconds);
-    thd->time_zone_used = true;
+//    thd->time_zone_used = true;
```
Comment (ihnorton) [inline mytile/mytile-metadata.cc:333]: remove

File: mytile/mytile.cc [hunk 2]
```diff
@@ -790,7 +790,7 @@ int tile::set_datetime_field(THD *thd, Field *field,
   MYSQL_TIME to;
   if (type != MYSQL_TIMESTAMP_DATE) {
     thd->variables.time_zone->gmt_sec_to_TIME(&to, seconds);
-    thd->time_zone_used = true;
+//    thd->time_zone_used = true;
```
Comment (ihnorton) [inline mytile/mytile.cc:793]: remove

**Comment by niclar** [comment]
Will this me merged anytime soon ?

**Comment by DimitrisStaratzis** [comment]
> Will this me merged anytime soon ?

Hi @niclar,
Yes we are planning for a release today

**Comment by DimitrisStaratzis** [comment]
Delaying because of nightly run failures in macOS

**Comment by DimitrisStaratzis** [comment]
The macOS nightly failures in this PR are unrelated to the changes made here. 
Proof is this rerun of the nightly in the master branch: https://github.com/TileDB-Inc/TileDB-MariaDB/issues/285

**Pull request merged**